apps/src/code-studio/initApp/project.js

```diff
@@ -185,6 +186,14 @@ var projects = module.exports = {
    * @returns {boolean} true if project has been reported enough times to
    *   exceed our threshold
    */
+  hasPrivacyProfanityViolation: function () {
```

Comment:
can just be `hasPrivacyProfanityViolation() {` here and elsewhere in this file I think